[1,"Error occurred during cleanup: Error: El proceso no tiene acceso al archivo porque est√° siendo utilizado por otro proceso. (os error 32): remove 'C:\\Users\\gduar\\AppData\\Local\\Temp\\quarto-sessionb6ed0d080ea886e1'\n"]
